### PR TITLE
fix: full name of app authors and added correct urls

### LINKED ISSUE #9087: [CAL-1754] App Store: Use the full name of the creator of the app and have it link to their Github?
Currently it's this… not the most reliable looking of sources.

This person has a Github profile with a full name:

Can we have it link to the Github and show it as:

* Published by [Shivam Kalra](https://github.com/shivamklr)

It looks better for us and it's a better credit for them

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1754](https://linear.app/calcom/issue/CAL-1754/app-store-use-the-full-name-of-the-creator-of-the-app-and-have-it-link)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
before we showed the github username

now we show the full name + link to github

CleanShot 2023-05-24 at 16 25 58@2x

### CODE REVIEW

File: packages/app-store/zoho-bigin/config.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "slug": "zoho-bigin",
   "type": "zoho-bigin_other_calendar",
   "logo": "zohobigin.svg",
-  "url": "https://cal.com/apps/zoho-bigin",
+  "url": "https://cal.com/",
   "variant": "other",
   "categories": ["other"],
-  "publisher": "Cal.com",
+  "publisher": "Cal.com, Inc.",
```
Comment (hariombalhara) [inline packages/app-store/zoho-bigin/config.json:10]: This is actually published by a contributor, will update it

File: packages/app-store/zohocrm/config.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "slug": "zohocrm",
   "type": "zohocrm_other_calendar",
   "logo": "icon.png",
-  "url": "https://cal.com/apps/zohocrm",
+  "url": "https://cal.com/",
   "variant": "other",
   "categories": ["other"],
-  "publisher": "Cal.com ",
+  "publisher": "Cal.com, Inc.",
```
Comment (hariombalhara) [inline packages/app-store/zohocrm/config.json:10]: It's also by a contributor

File: packages/app-store/webex/config.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "type": "webex_video",
   "imageSrc": "/icon.ico",
   "logo": "/icon.ico",
-  "url": "https://cal.com/apps/webex",
+  "url": "https://cal.com",
   "variant": "conferencing",
   "categories": ["video"],
   "publisher": "Cal.com, Inc.",
```
Comment (hariombalhara) [inline packages/app-store/webex/config.json:12]: It's also by a contributor

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Will fix. Approve and merge then.

File: packages/app-store/webex/config.json (hunk 3 above)
Comment (PeerRich) [reply packages/app-store/webex/config.json:12]: thank you @hariombalhara for looking out!

File: packages/app-store/webex/config.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "type": "webex_video",
   "imageSrc": "/icon.ico",
   "logo": "/icon.ico",
-  "url": "https://cal.com/apps/webex",
+  "url": "https://cal.com",
```
Comment (hariombalhara) [inline packages/app-store/webex/config.json:9]:
```suggestion
  "url": "https://github.com/aar2dee2",
```

File: packages/app-store/webex/config.json (hunk 3 above)
Comment (hariombalhara) [inline packages/app-store/webex/config.json:12]:
```suggestion
  "publisher": "aar2dee2",
```

File: packages/app-store/zoho-bigin/config.json (hunk 1 above)
Comment (hariombalhara) [inline packages/app-store/zoho-bigin/config.json:10]:
```suggestion
  "publisher": "Shane Maglangit",
```

File: packages/app-store/zoho-bigin/config.json [hunk 5]
```diff
@@ -4,10 +4,10 @@
   "slug": "zoho-bigin",
   "type": "zoho-bigin_other_calendar",
   "logo": "zohobigin.svg",
-  "url": "https://cal.com/apps/zoho-bigin",
+  "url": "https://cal.com/",
```
Comment (hariombalhara) [inline packages/app-store/zoho-bigin/config.json:7]:
```suggestion
  "url": "https://github.com/ShaneMaglangit",
```

File: packages/app-store/zohocrm/config.json [hunk 6]
```diff
@@ -4,10 +4,10 @@
   "slug": "zohocrm",
   "type": "zohocrm_other_calendar",
   "logo": "icon.png",
-  "url": "https://cal.com/apps/zohocrm",
+  "url": "https://cal.com/",
```
Comment (hariombalhara) [inline packages/app-store/zohocrm/config.json:7]:
```suggestion
  "url": "https://github.com/jatinsandilya",
```

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**